.github/workflows/tests.yml

```diff
@@ -0,0 +1,22 @@
+name: Tests
+on:
+  pull_request:
```

Comment:
suggest adding https://github.com/near-cli-rs/interactive-clap/blob/master/.github/workflows/release-plz.yml#L8-L10 here and in `.github/workflows/code_style.yml` too